### ISSUE TITLE
Different filters in individual Material lists

### ISSUE THREAD

**Issue by phlnx** [EDDiscovery/EDDiscovery#2943]
I had the idea of creating a new panel with multiple split sub-panels, each of which would have a Materials list filtered for a specific engineer needs (For example the FSD Increased Range would have Phosphorus, Chemical Manipulators, Arsenic, etc; the Dirty Thrusters would have Mechanical Components, Cadmium,Cracked Industrial Firmware, etc). However I noticed that when I apply a filter to one of the sub-panels, the other Materials sub-panels would refresh and essentially replicate the exact filter I just confirmed, as a result I cannot have different filters on different Materials sub-panels. Is there a setting I can operate on or is this a limitation of the software?

Thank you for your attention, let me know if you need further details.

**Comment by robbyxp1** [comment]
I'll check. It should not be working this way. thanks

**Comment by robbyxp1** [comment]
I can't reproduce.

Here is two engineering panels, in a splitter...

![image](https://user-images.githubusercontent.com/6573992/98524902-ae8b2180-226f-11eb-995d-26cfa693effa.png)

I've filtered the top on material = worn shield emiiters, the second on carbon. It appears to work. 11.7.5.

Any more details, like what exact panels you are using (I presume the engineering one)..

**Comment by phlnx** [comment]
I apologise for having been imprecise. These are the steps I used.

1. Create new Grid or Splitter tab
2. Inside the Grid/Splitter tab, add the Materials tab, two or three times.
3. Inside the Grid/Splitter tab, Choose the Materials tab #1, filter, click on None then on any number of materials
4. Repeat for other Materials tabs so that they all have different filters.
5. Quit EDDiscovery
6. Start EDDiscovery. All the Materials element in the Grid/Splitter will have the same filter setting.

I can confirm that I am running 11.7.5.0.

I wasn't aware of the Engineering tab, which seems to do part of the work I had set out to do by using the Materials tab.

As a side question, is it possible to hide columns?

Thank you for your help.

**Comment by robbyxp1** [comment]
Still no luck!

![image](https://user-images.githubusercontent.com/6573992/98535723-80610e00-227e-11eb-8ff9-d255bc4047a2.png)

Its holding its filter for raw, manu, encoded.. tried a few other combos.

Is this the setup your using?

There is a shopping list tab as well..

**Comment by phlnx** [comment]
I must confess you are right, I tried again with a brand new tab, and it didn't behave the way I reported, rather it behaved as I would expect. I must offer my apologies for having wasted your time.

However I noticed that if I remove a splitter tab, then I add a brand new splitter tab, I get the previous tab, which I had previously removed, with all of its configuration and everything. 

As for my previous question, is it possible to remove certain columns from such lists, like Materials? Ideally I only need Name and Number and the other columns aren't really useful to me. However resizing them away is bothersome.

Thank you.
